pysat/tests/test_instrument.py

```diff
@@ -115,46 +191,110 @@ def test_next_load_default(self):
         self.ref_time = dt.datetime(2008, 1, 1)
         self.testInst.next()
         self.out = self.testInst.index[0]
-        self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
         assert self.out == self.ref_time
+        self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
+        assert (self.out == self.testInst.date)
 
     def test_prev_load_default(self):
         """Test if last day is loaded by default when first invoking .prev.
         """
         self.ref_time = dt.datetime(2010, 12, 31)
         self.testInst.prev()
         self.out = self.testInst.index[0]
-        self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
         assert self.out == self.ref_time
+        self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
+        assert (self.out == self.testInst.date)
 
-    def test_basic_fid_instrument_load(self):
-        """Test if first day is loaded by default when first invoking .next.
+    def test_next_load_bad_start_file(self):
+        """Test Error if trying to iterate when on a file not in iteration list
         """
         self.ref_time = dt.datetime(2008, 1, 1)
-        self.testInst.load(fid=0)
-        self.out = self.testInst.index[0]
-        self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
-        assert (self.out == self.ref_time)
-        assert (self.out == self.testInst.date)
+        self.testInst.load(fname=self.testInst.files[1])
+        # set new bounds that doesn't include this date
+        self.testInst.bounds = (self.testInst.files[0], self.testInst.files[20],
+                                2, 1)
+        with pytest.raises(StopIteration):
+            self.testInst.next()
+
+    def test_prev_load_bad_start_file(self):
+        """Test Error if trying to iterate when on a file not in iteration list
+        """
+        self.ref_time = dt.datetime(2008, 1, 1)
```

Comment:
Does this test not work for 1/1/2009?  Also applies to the tests below.  It seems like you wouldn't need to alter the date if you adjusted the load and bounds, which are already being reset here.